seed/static/seed/partials/mapping.html

```diff
@@ -83,23 +83,23 @@
     <div class="section_header_container has_no_border" ng-hide="review_mappings">
         <div class="section_header fixed_height">
             <div class="left section_action_container">
-                <button type="button" class="btn btn-primary" ng-hide="import_file.matching_done && isValidCycle" ng-disabled="check_reset_mappings() || import_file.source_type == 'BuildingSync Raw'" ng-click="reset_mappings()" translate>Copy Data File Headers directly to SEED Headers</button>
+                <button type="button" class="btn btn-primary" ng-hide="import_file.matching_done && isValidCycle" ng-disabled="check_reset_mappings() || mappingBuildingSync" ng-click="reset_mappings()" translate>Copy Data File Headers directly to SEED Headers</button>
             </div>
             <div class="right section_action_container section_action_btn">
-                <button type="button" class="pull-right btn btn-primary mapping-button" ng-disabled="check_fields()" ng-click="remap_buildings()" ng-hide="import_file.matching_done" translate>Map Your Data</button>
+                <button type="button" class="pull-right btn btn-primary mapping-button" ng-disabled="check_fields() || is_buildingsync_and_preset_not_ok()" ng-click="remap_buildings()" ng-hide="import_file.matching_done" translate>Map Your Data</button>
```

Comment:
Do the same for the bottom button:
<img width="988" alt="Screen Shot 2020-05-18 at 2 26 43 AM" src="https://user-images.githubusercontent.com/30608004/82240781-37945200-98f8-11ea-95c5-5907aebc60cc.png">